gwtmockito/src/main/java/com/google/gwtmockito/impl/ReturnsCustomMocks.java

```diff
@@ -42,6 +43,8 @@ public Object answer(InvocationOnMock invocation) throws Throwable {
     } else if (invocation.getMock() instanceof Element && methodName.equals("getTagName")) {
       String className = invocation.getMock().getClass().getSimpleName();
       return className.substring(0, className.indexOf("Element")).toLowerCase();
+	} else if (invocation.getMock() instanceof InputElement && methodName.equals("getType")) {
+		return "text";
```

Comment:
Could you add a test for this? It can go next to the similar one at https://github.com/google/gwtmockito/blob/master/gwtmockito/src/test/java/com/google/gwtmockito/GwtMockitoTest.java#L587